### ISSUE TITLE
Working demo with Google Analytics and IE Support

### ISSUE THREAD

**Issue by Allmedialab** [thany/gdpr-cookie#3]
Hi,
Thanks for the nice plugin!
Would it be possible to create a working demo with as an example Google Analytics and included IE Support?
I'm sure more people are able to understand your script (like me!) and use it.
Regards & Thanks,
David

**Comment by tmdevde** [comment]
I'm using it like this:

Init the GDPR Cookie Script:
```
$.gdprcookie.init({
    title: "Cookies & Datenschutz",
    message: "Wir benutzen Cookies <small>🍪</small> um Ihnen das bestmögliche Erlebnis auf dieser Website bieten zu können.",
    subtitle: "Cookies bearbeiten",
    acceptBtnLabel: "Akzeptieren",
    advancedBtnLabel: "Cookies einstellen",
    cookieTypes: [
        {
            'type': 'Essentiell',
            'value': 'essential',
            'description': 'Essentielle Cookies dieser Website.',
        },
        {
            'type': 'Analytics',
            'value': 'analytics',
            'description': 'Cookies im Bezug auf Seitenbesuche, Browserarten, ect.',
        }]
});
```

When the user clicks the accept button run Google Analytics script to activate or deactivate Google Analytics
```
$("body").on('gdpr:accept', function() {
    if ($.gdprcookie.preference("analytics") === true) { gaOptIn(); } else { gaOptout(); }
});
```
Put a link on your side with the id "cookies" so the user can change his mind when he clicks on it.
```
$("#cookies").click(function() {
    $.gdprcookie.display();
});
```

The Google Analytics Script
```
<script type="text/javascript">
  var gaProperty = 'UA-YOUR-ID';
  var disableStr = 'ga-disable-' + gaProperty;
  if (document.cookie.indexOf(disableStr + '=true') > -1) {
    window[disableStr] = true;
   }
   function gaOptout() {
     document.cookie = disableStr + '=true; expires=Thu, 31 Dec 2099 23:59:59 UTC;path=/';
     window[disableStr] = true;
   }
   function gaOptIn() {
     document.cookie = disableStr + '=false; expires=Thu, 31 Dec 2099 23:59:59 UTC;path=/';
     window[disableStr] = false;
   }
</script>
```

Hope this is some kind of help.

**Comment by Allmedialab** [comment]
Hi, 
Thanks very much! I tried your suggestion, but unfortunately it still loads the Analytics cookies on page load.
Regards,
David

**Comment by tmdevde** [comment]
Hmm it's working for me. Did you test it with Google Analytics Realtime Visitors? For me Google Analytics isn't tracking any user data when i optOut, and that's what i want.

**Comment by tmdevde** [comment]
I send you an email.

**Comment by Allmedialab** [comment]
Do you need my email-address?

**Comment by tmdevde** [comment]
I send you one to the address i found in your github profile but you can post it here too if you want.

**Comment by Allmedialab** [comment]
info@allmedialab.nl

**Comment by thany** [comment]
@Allmedialab Afaik, my script already supports IE11. I haven't tested earlier versions. Also thanks for you kind words 😀
As for a demo, there's not much demo to be had, because as it says in the documentation, I'm personally very much biased towards putting the rendering of cookie-sensetive elements on the server.

@tmdevde I'm not much of a supporter of this method personally, because even if the visitor unchecks analytics cookies, it will *still* render and execute Analytics. The way I see it, is when the visitor unchecks analytics, you shouldn't be tracking them at all anymore, and that means disabling GA alltogether.

Furthermore, the opt-in and opt-out feature in GA cannot fully be trusted because its a third party, and, let's be honest, it's Google. They will track your visitors with or without cookies. As a site owner, you are still responsible, so it's better to be safe than sorry. Sure, your analytics will be skewed, but you can also be proud not to be tracking your visitors if they request not to be tracked 😙

However, if you can't change anything on the server and/or don't have a template engine to put some kind of if-statement in to conditionally render blocks of trackers, your `ga-disable-*` setting might be the next best thing.

TL;DR
Implementation is up to you. My plugin merely provides a front-end to facilitate asking for visitor's consent.

**Comment by tmdevde** [comment]
@thany this is the official gdpr code to disable google analytics tracking. yes the code will be rendered but analytics won't track the user anymore. i tested it by myself. if your render the code or not doesnt matter. only the parameter and only the tracking of personal data is important. you can render as much GA code as you want, the only important think is: will there be data transfered and saved to identify a user. if not nobody cares. but the user needs the option to decide if the owner wants to.

i don't want to discuss this here much but the whole script makes no sense in your case because noone will accept any cookies if he doesn't need to. please have a look at the law again.
